### ISSUE TITLE
Use alternate method for Form Data entry

### ISSUE THREAD

**Issue by MarsRomer** [bcgov/CONN-CCBC-portal#2142]
As a analyst 
I want to upload the form data without an intake
So that I can handle exceptions to the business process

**Description:**
We need an alternate route for entering form data to align with new business processes on behalf of ISP 

# Acceptance Criteria:
- [x] Readme file with step by step instructions of how these applications get uploaded - good for re-doing and re-tracing (for audit)
- [x] Internal user can create an application using custom URL in prod using Basic BCeID
- [x] Applications can be created at any time and multiples times in the next few months without affecting future or ongoing intakes
- [x] CCBC ID will be CCBC-99000##
- [x] When analyst sees the application in the portal it should have status = received (Developer runs a separate job to convert status from Submitted to Received on demand (manually run))
- [x] ISP on whose behalf the application is submitted can see the application on their dashboard, they can do all the same actions on the dashboard as an application they would have submitted themselves - i.e. view, withdraw (coming in another ticket), submit RFI files.

**Comment by MarsRomer** [comment]
blocked by #2199

**Comment by MarsRomer** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/CCBC---Delivery-Board-61c0c1204c8bcb001491c5f3/issues/bcgov/CONN-CCBC-portal/2142?planning-poker) with Zenhub @AntBush @rafasdc

**Comment by AntBush** [comment]
Link to the proof of concept pull request Vlad created for this ticket: 

https://github.com/bcgov/CONN-CCBC-portal/pull/2247

**Comment by MarsRomer** [comment]
@AntBush This works well!  The only thing is that  'Create new application' button is a bit glitchy, I think you had the same problem during the demo.  I can put this to Sprint Done so we can release to prod and create a bug ticket for it if that's ok.  The multiple tabs modal did pop up unnecessarily like it did for you too but that's a different ticket
